collect_app/src/main/java/org/odk/collect/android/geo/MapboxMapFragment.java

```diff
@@ -212,6 +217,18 @@ public class MapboxMapFragment extends org.odk.collect.android.geo.mapboxsdk.Map
     @Override public void onAttach(@NonNull Context context) {
         super.onAttach(context);
         DaggerUtils.getComponent(context).inject(this);
+
+        MapboxMapConfigurator configurator = new MapboxMapConfigurator(
```

Comment:
The same here it is constructed in the same way in two places so maybe it should be moved to `MapboxMapConfigurator`.